zetaclient/chains/evm/signer/signer.go

```diff
@@ -529,22 +538,21 @@ func (signer *Signer) BroadcastOutbound(
 	if tx == nil {
 		logger.Warn().Msgf("BroadcastOutbound: no tx to broadcast %s", cctx.Index)
 	}
-	// Try to broadcast transaction
+
+	// broadcast transaction
 	if tx != nil {
 		outboundHash := tx.Hash().Hex()
-		logger.Info().
-			Msgf("on chain %s nonce %d, outboundHash %s signer %s", signer.chain, cctx.GetCurrentOutboundParam().TssNonce, outboundHash, myID)
-		//if len(signers) == 0 || myid == signers[send.OutboundParams.Broadcaster] || myid == signers[int(send.OutboundParams.Broadcaster+1)%len(signers)] {
+		logger.Info().Msgf("BroadcastOutbound: broadcasting tx %s on chain %d nonce %d signer %s",
```

Comment:
I am wondering if we should move this to debug. 
The broadcasted log is already in the info. Do we need both ?